worker/storageprovisioner/storageprovisioner.go

```diff
@@ -16,6 +18,13 @@ import (
 	"github.com/juju/juju/worker/storageprovisioner/internal/schedule"
 )
 
+var (
+	// defaultDependantChangesTimeout is the default timeout for waiting for any
+	// dependant changes to occur before proceeding with a storage provisioner.
+	// This is a variable so it can be overridden in tests (sigh).
+	defaultDependantChangesTimeout = time.Second
```

Comment:
One would tend to use "dependant" as a noun and "dependent" as an adjective. I think the latter is more suitable here.